src/queueserver.ts

```diff
@@ -450,3 +450,24 @@ export const modifyQueue = async(op: QueueOps): Promise<IPlanModify> => {
     console.log(res);
     return res.data;
 }
+
+export const incrementPosition = async(item1_uid: string, item2_uid: string): Promise<IPlanModify> => {
+    const res = await axiosInstance.post(`/queue/item/move`,
```

Comment:
Why are the backticks used here? I think they are needed to expand the variables, as in the case above.

Reply:
Ohh.. I didn't know the meaning of the backticks. I just copy/pasted.

Reply:
Fixed

Reply:
Thanks! It's something that @cryos discovered some time ago when we discussed the equivalent of Python's f-strings.